apps/frontend/src/app/PageCharacter/CustomMultiTarget.tsx

```diff
@@ -143,14 +152,15 @@ export function CustomMultiTargetButton() {
   }, [origUIData, teamData])
 
   return <Suspense fallback={<Skeleton variant="rectangular" height="100%" width={100} />}>
-    <Button color="info" onClick={onShow} startIcon={<Settings />}>{t`multiTarget.title`}</Button>
+    <Button color="info" onClick={onShow} startIcon={<SettingsIcon />}>{t`multiTarget.title`}</Button>
     <DataContext.Provider value={dataContextObj}>
       <ModalWrapper open={show} onClose={onClose} containerProps={{ sx: { overflow: "visible" } }}>
         <CardDark>
           <CardContent sx={{ display: "flex", flexDirection: "column", gap: 1 }}>
-            <Box display="flex" gap={1} justifyContent={"space-between"}>
+            <Box display="flex" gap={1} alignItems="center">
               <Typography variant="h6">{t`multiTarget.title`}</Typography>
-              <CloseButton onClick={onClose} />
+              <InfoTooltip title={<Trans t={t} i18nKey="multiTarget.info">Note: Community created custom Multi-Optimization Targets can be found within the <a href={process.env.NX_URL_DISCORD_GO} target="_blank" rel="noreferrer" style={{ color: "white" }}>GO Discord</a> or <a href={process.env.NX_URL_DISCORD_KQM} target="_blank" rel="noreferrer" style={{ color: "white" }}>KQM Discord</a>, however the validity of such configurations cannot be guaranteed.\n\nIt is the responsibility of the user to confirm the accuracy for their own use case.</Trans>} />
```

Comment:
Wrap text here in `typography`, it looks too smol. Also, instead of redirecting to KQM, link to their [multi-opt collection doc](https://docs.google.com/document/d/12hrPJ3MrAF7t03gyiiVtV5uI_rUKlVlbEwl9Lf2esIQ/edit?usp=sharing)